core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -266,6 +267,36 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
     }
   }
 
+  shortenName(name: string): string {
+    if (name.length > 20) {
+      let spaceIndex = 0;
+      for (let i = 0; i < name.length; i++) {
```

Comment:
why do you need to loop over the name? can't you fix this by specifying the header width to a fixed size?

Reply:
please adjust the size using css not this way, also would the pencil icon for modifying the name disappear like this?